css-view-transitions-2/Overview.bs

```diff
@@ -983,6 +1083,67 @@ When capturing the old or new state for an element, perform the following steps
 	Note: This is written in a monkey-patch manner, and will be merged into the algorithm once the L1 spec graduates.
 </div>
 
+## Capturing and applying 'view-transition-group' ## {#vt-group-algorithm}
+
+### Compute the nearest containing 'view-transition-group' ### {#vt-group-nearest-contain}
+<div algorithm="nearest containing group name">
+
+To get the <dfn>nearest containing group name</dfn> for an {{Element}} |element|, perform the following steps:
+
+	1. Let |nearestAncestorWithContain| be |element|'s nearest ancestor whose 'view-transition-name' [=computed value=] is not ''view-transition-name/none'' and whose 'view-transition-group' [=computed value=] is ''view-transition-group/contain''.
+	1. If |nearestAncestorWithContain| is null, return ''view-transition-group/none''.
+	1. Otherwise, return |nearestAncestorWithContain|'s 'view-transition-name' [=computed value=].
+</div>
+
+### Compute the old 'view-transition-group' ### {#vt-group-capture-old}
+<div algorithm="additional old capture steps (group)">
+When  [[css-view-transitions-1#capture-old-state-algorithm|capturing the old state for an element]], perform the following steps given a [=captured element=] |capturedElement|, and an [=element=] |element|:
+
+	1. Set |capturedElement|'s [=captured element/containing group name=] to the [=used value=] of |element|'s ''view-transition-group''.
+</div>
+
+### Compute the new 'view-transition-group' ### {#vt-group-capture-new}
+<div algorithm="additional new capture steps (group)">
+When [[css-view-transitions-1#capture-new-state-algorithm|capturing the new state for an element]], perform the following steps given a [=captured element=] |capturedElement| and an [=element=] |element|:
+
+	1. Set |capturedElement|'s [=captured element/containing group name=] to the [=used value=] of |element|'s 'view-transition-group'.
+</div>
+
+### Reparent a ''::view-transition-group()'' to its specified containing group ### {#vt-group-reparent}
+<div algorithm="additional pseudo-element setup steps (group)">
+When [[css-view-transitions-1#setup-transition-pseudo-elements-algorithm|setting up the transition pseudo-element]] for a [=captured element=] |capturedElement|, given a |transitionName| and a |transition|:
+
+	1. Let |containingGroupName| be |capturedElement|'s [=containing group name=].
+	1. If |containingGroupName| is not null, then:
+
+		1. Let |groupContainerElement| be |transition|'s [=ViewTransition/named elements=][|containingGroupName|].
+
+		1. Let |group| be the ''::view-transition-group()'',
+			whose [=view transition name=] is set to |transitionName|.
+
+		1. Let |parentGroup| be the ''::view-transition-group()'',
+			whose [=view transition name=] is set to |containingGroupName|.
+
+		1. Append |group| to |parentGroup|.
+
+		1. When setting the animation keyframes given |transform|, [=multiply=] |transform| by the inverse matrix of |groupContainerElement|'s [=old transform=].
```

Comment:
When we get to merge the specs, would be nice to rename this to "transform from snapshot containing block".